### DISCUSSION TITLE
如何支持xmake和trybuild两种构建系统同时使用

### DISCUSSION THREAD

**Discussion by Ekkone** [xmake-io/xmake#1698] [Q&A]
如何在同一个文件夹下，既可以使用编写的xmake.lua进行构建，又可以使用trybuild模式使用cmake进行构建？
目前只要文件夹内存在xmake.lua，执行xmake就默认直接构建，能否通过什么配置命令来选择使用xmake或是其他trybuild模式呢？

**Comment by waruqi** [comment]
不支持
